.github/workflows/image-arm.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         git fetch --prune --unshallow
     - id: set-matrix
       run: |
-          content=`cat ./.github/flavors-arm.json`
+          content=`cat ./.github/flavors.json | jq -r 'map(select(.arch == "arm64" and .variant == "standard" and .model != "generic"))'`
```

Comment:
This is `get-core-matrix`. I think this should have been `.variant == "core"`.

Reply:
the reusable workflow that uses the matrix has the variant hardcoded to "core" anyway, that's why this worked. Will fix anyway so that we pick the right flavors.